src/main/java/org/opensearch/ad/task/ADBatchTaskRunner.java

```diff
@@ -792,12 +791,6 @@ public void startADBatchTaskOnWorkerNode(
                 adTaskManager.cleanDetectorCache(adTask, transportService, () -> handleException(adTask, e));
             } else {
                 adTaskManager.entityTaskDone(adTask, e, transportService);
-                threadPool
```

Comment:
So if there is an exception, we will not start the remaining entity task and just stop?

Reply:
talked with you offline.  Summary for others: entityTaskDone will send a msg from the worker node to the coordinating node.  The coordinating node will put the failing task to the end of running queue and start a new task lane (may fail if others steal the lane in between the messaging) for the current head entity task on the queue.  Without removing the code, the worker node will restart the failing task (max retries: 3).  When the coordinating node comes to check, it finds the lane is already used and won't start new task.  After removing the code, the coordinating node will sends a new task to start.  The benefit of removing the code is that the failing task might be failing quickly again.  It is better to try other tasks before exhausting the 3 retries.